sdk/src/main/java/com/ingresse/sdk/model/request/HighlightMethod.kt

```diff
@@ -0,0 +1,8 @@
+package com.ingresse.sdk.model.request
+
+import com.google.gson.annotations.SerializedName
+
+enum class HighlightMethod {
```

Comment:
Esse enum é realmente necessário?

Reply:
Fiz um mapeamento para quando for necessário usarmos esse filtro algum dia já termos uma classe enum preparada e uma indicação no request de que é possível a utilização do mesmo. Agora não estamos usando o "banner" mas existe essa possibilidade, para o carrossel por exemplo. Alinhei com a Tah e achei melhor usar um enum.